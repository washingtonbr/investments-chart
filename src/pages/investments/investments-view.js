import React from 'react';
import DataFilter from '../../components/data-filter';
import PeriodInfo from '../../components/period-info';
import InvestmentsChart from '../../components/investments-chart';
import { Container, FilterRow } from './investments-styles';

export default function InvestmentView({
  labels,
  data,
}) {
  return (
    <Container>
      <FilterRow>
        <DataFilter />
      </FilterRow>

      <PeriodInfo />

      <InvestmentsChart
        labels={labels}
        data={data}
      />
    </Container>
  );
}
