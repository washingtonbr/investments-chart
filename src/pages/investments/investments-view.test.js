import React from 'react';
import renderer from 'react-test-renderer';
import InvestmentsView from './investments-view';
import { mockLabels, mockData } from './investments-view.mock';

jest.mock('../../utils/date');

describe('InvestmentsView', () => {
  xit('renders correctly', () => {
    const tree = renderer
      .create(
        <InvestmentsView
          labels={mockLabels}
          data={mockData}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
