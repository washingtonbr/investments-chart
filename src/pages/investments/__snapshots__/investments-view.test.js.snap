// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvestmentsView renders correctly 1`] = `
<div
  className="sc-AxhUy bnBzHX"
>
  <div
    className="sc-AxgMl jDCFvF"
  >
    <div
      className="sc-AxjAm bDujyt"
    >
      <select
        className="sc-AxirZ iEPwaG"
        onChange={[Function]}
        value={-1}
      />
    </div>
  </div>
  <canvas
    height={55}
    width={100}
  />
</div>
`;
